lib/ui/pages/account_dashboard.dart

```diff
@@ -30,62 +32,105 @@ class AccountDashboard extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return SafeArea(
-      child: RefreshIndicator(
-        onRefresh: () async {
-          Get.find<AccountDashboardController>().onRefresh();
-        },
-        child: SingleChildScrollView(
-          physics: const AlwaysScrollableScrollPhysics(),
-          child: Container(
-            padding: const EdgeInsets.symmetric(
-              horizontal: 20,
+    return GetBuilder<AccountDashboardController>(builder: (value) {
+      if (value.isLoading) {
+        return Scaffold(
+          body: Container(
+            child: Center(
+              child: CircularProgressIndicator(
+                valueColor: AlwaysStoppedAnimation<Color>(LightColor.yellow2),
+              ),
             ),
-            height: Get.height,
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: <Widget>[
-                const SizedBox(
-                  height: 35,
-                ),
-                AccountDashboardAppBar(() {
-                  _showAccountChoosingBottomSheet();
-                }),
-                const SizedBox(
-                  height: 50,
-                ),
-                const TitleText(
-                  text: 'Operations',
-                ),
-                const SizedBox(
-                  height: 10,
-                ),
-                Operations(),
-                const SizedBox(
-                  height: 40,
+          ),
+        );
+      } else {
+        if (value.noAccounts == false) {
+          return SafeArea(
+            child: RefreshIndicator(
+              onRefresh: () async {
+                Get.find<AccountDashboardController>().onRefresh();
+              },
+              child: SingleChildScrollView(
+                physics: const AlwaysScrollableScrollPhysics(),
+                child: Container(
+                  padding: const EdgeInsets.symmetric(
+                    horizontal: 20,
+                  ),
+                  // height: Get.height,
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      const Padding(
+                        padding: EdgeInsets.fromLTRB(10, 30, 0, 30),
+                        child: TitleText(
+                          text: 'Accounts',
+                          fontSize: 20,
+                        ),
+                      ),
+                      AccountDashboardAppBar(() {
+                        _showAccountChoosingBottomSheet();
+                      }),
+                      const SizedBox(
```

Comment:
can you tidy up the code over here by not using trailing commas for simple classes